### PR TITLE
Heading feature 

### PR DESCRIPTION
**Pull request by ntazelo**
**HELLO WORLD PROJECT**

     In the Hello world project I added : 

       1. An HTML file with an h1 
       2. An CSS file with a style on the h1
       3. A text.MD  file that is not tracked by .gitignore file
       4. A linter to remove linting issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,92 @@
-# Hello-world
```
Comment (SajjadAhmad14) [inline README.md:1]: Please add a professional README. A README file should have a project description and steps to run the project locally. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,92 @@
...
-A hello world website
+# HTML & CSS3 Course
```
Comment (SajjadAhmad14) [inline README.md]: Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and follow best practices to write professional commit messages.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Eloi Ntazama , 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (ntazelo) [reply README.md:1]: Thank you for the review I added a new README file.

File: README.md (hunk 2 above)
Comment (ntazelo) [reply README.md]: Thank you for the review I capitalized the commit text on my last commit.

**Review by GzimAsani — APPROVED** [review]
# Hello @ntazelo 

## Thank you for implementing the required change asked by the previous reviewer. Your project is **ACCEPTED :green_circle:** and I wish you best of luck in your next projects :+1: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:
 
 Cheers and Happy coding!

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**